keps/sig-api-machinery/5073-declarative-validation-with-validation-gen/README.md

```diff
@@ -708,53 +709,37 @@ The below rules are currently implemented or are very similar to an existing val
     N/A
    </td>
   </tr>
-</table>
-
-The below rules are not currently implemented in the [validation-gen prototype](https://github.com/jpbetz/kubernetes/tree/validation-gen) so the exact syntax is still WIP
-
-<table>
-  <tr>
-   <td>
-    <strong>Type of validation</strong>
-   </td>
-   <td>
-    <strong>IDL tag</strong>
-   </td>
-   <td>
-    <strong>Relative OpenAPI validation field</strong>
-   </td>
-  </tr>
   <tr>
    <td>
-    regex matches
+    union member (Discriminated/Non-Discriminated)
    </td>
    <td>
-    `+k8s:pattern`
+    +k8s:unionMember={"union": ""}
    </td>
    <td>
-    `pattern`
+    x-kubernetes-unions
    </td>
   </tr>
   <tr>
    <td>
-    cross field validation
+    union discriminator
    </td>
    <td>
-    `TBD
+    +k8s:unionDiscriminator={"union": ""}
    </td>
    <td>
-    `x-kubernetes-validations`
+    x-kubernetes-unions
    </td>
   </tr>
   <tr>
    <td>
-    <a href="https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#transition-rules">transition rules</a>
+    immutable value
    </td>
    <td>
-    `TBD
+    +k8s:immutable
    </td>
    <td>
-    `x-kubernetes-validations`
+    x-kubernetes-validations (CEL: !has(oldSelf...) || self... == oldSelf...)
```

Comment:
Accidentally left CEL here?  If we're trying to avoid it, we don't need it here right?

Reply:
This intention of the text there was to show the OpenAPI equivalent for +k8s:immutable which would be using x-kubernetes-validations with a CEL expression similar to that shown.  I can remove this text, just wanted to note the current intent which is aligned with not using CEL in v1.34.  This column is to note how some declarative validation rules map-to/could-be-output as OpenAPI rules.

Reply:
I've removed this now, the immutability KEP Update already added a row related to this that did not reference CEL.